tests/kolainst/destructive/soft-reboot.sh

```diff
@@ -23,21 +24,24 @@ case "${AUTOPKGTEST_REBOOT_MARK:-}" in
   # Deploy the new commit normally first
   ostree admin deploy --stage soft-reboot-test
 
-  assert_status_jq '.deployments[0].pending' '.deployments[0].["soft-reboot-target"] | not'
+  assert_status_jq '.deployments[0].staged' '.deployments[0].["soft-reboot-target"] | not'
 
   # Test prepare-soft-reboot command
   echo "Testing prepare-soft-reboot..."
   ostree admin prepare-soft-reboot 0
 
   # Test human readable format
   ostree admin status > status.txt
-  assert_file_has_content_literal status.txt '(pending) (soft-reboot)'
+  assert_file_has_content_literal status.txt '(staged) (soft-reboot)'
```

Comment:
Note this one is important! We now properly support staging and soft rebooting together.